themes/default/data/registry/packages/impart.yaml

```diff
@@ -9,6 +9,6 @@ package_status: public_preview
 publisher: Impart Security
 repo_url: https://github.com/impart-security/pulumi-impart
 schema_file_path: provider/cmd/pulumi-resource-impart/schema.json
-title: Impart
-updated_on: 1685834716
-version: v0.1.4
+title: Impart Security
```

Comment:
@sbzimpart i believe this is because you are changing the title here, instead you are going to want to add a display name and if you are using any generation automation you will want to re-run that.

Reply:
actually im not sure, @sean1588 thoughts?

Reply:
@susanev my guess the previes only build the docs for these 2 packages:

https://github.com/pulumi/registry/blob/4a8b1786c72666d6888a6515790340c96bc75c87/scripts/ci/build.sh#L14C1-L17

Reply:
This is happening for the other pr as well. In this case Astra is not opening up. It also has some doc change. 
https://github.com/pulumi/registry/pull/2639
If that's expected maybe this pr should be merged.

Reply:
@ringods but thats only for the api docs not the overview and install pages

Reply:
@susanev can you please close this pr unless there a fix for it?
I pushed another version where I reverted the display name / title changes. Will try to set them at another time once other things are take care of.